### PR TITLE
Recommend using bundle for running docs locally

### PR DESCRIPTION
**Pull request by hkdobrev**
Closes https://github.com/twbs/bootstrap/issues/17493

Continuing from #17227.

### CODE REVIEW

**Comment by XhmikosR** [comment]
Please update the docs too.

**Comment by hkdobrev** [comment]
@XhmikosR Updated the docs as well. I hope the language is clear, any suggestions welcome.

**Comment by XhmikosR** [comment]
It seems inconsistent. We should use the same stuff we mention in README.md and in docs.

/CC @cvrebert

**Comment by cvrebert** [comment]
Agreed. The README should use the more thorough wording that @hkdobrev used in step 4 in build-tools.md

**Comment by XhmikosR** [comment]
So, @hkdobrev can you take care of my previous comment?

**Comment by hkdobrev** [comment]
@XhmikosR @cvrebert Sorry for the delay, I've updated the readme. Cheers!

File: README.md [hunk 1]
```diff
@@ -66,11 +66,9 @@ Bootstrap's documentation, included in this repo in the root directory, is built
 
 ### Running documentation locally
 
-1. If necessary, [install Jekyll](http://jekyllrb.com/docs/installation) (requires v2.5.x).
-  - **Windows users:** Read [this unofficial guide](http://jekyll-windows.juthilo.com/) to get Jekyll up and running without problems.
-2. Install the Ruby-based syntax highlighter, [Rouge](https://github.com/jneen/rouge), with `gem install rouge`.
-3. From the root `/bootstrap` directory, run `jekyll serve` in the command line.
-4. Open <http://localhost:9001> in your browser, and voilà.
+1. Run through the [tooling setup](https://github.com/twbs/bootstrap/blob/master/docs/getting-started/build-tools#tooling-setup) to install Jekyll (the site builder) and other Ruby dependencies with `bundle install`.
+2. From the root `/bootstrap` directory, run `jekyll serve` in the command line.
```
Comment (XhmikosR) [inline README.md]: `bundle exec jekyll serve`

Comment (hkdobrev) [inline README.md]: :+1: Updated.

**Comment by XhmikosR** [comment]
@cvrebert: thoughts now?

**Comment by mdo** [comment]
Looks good to me FWIW.

**Pull request merged**